samples/client/petstore/python-experimental/test_python2.sh

```diff
@@ -1,33 +0,0 @@
-#!/bin/bash
```

Comment:
We need this file for testing, please add it back in

Reply:
Done

Reply:
Can you resolve this conversation?